tests/nonsmoke/functional/CompileTests/OpenMP_tests/Makefile.am

```diff
@@ -308,7 +308,12 @@ REX_C_TESTCODES_REQUIRED_TO_PASS = \
 	task_underIf.c \
 	taskgroup.c \
 	taskloop.c \
+	task_wait.c
```

Comment:
It seems because of missing the backslash symbols, any files after task_wait.c are not tested. Could you check again?

Reply:
please rename the file to taskwait.c since taskwait is the actual directive name.

Reply:
Both items are fixed.